api/user/login.py

```
@@ -3,38 +3,38 @@
 #   FUNCTION
 #
 
 # FUNCTION -> DECLARATION
-def output(request: object) -> object:
+def output(request: HttpRequest) -> HttpResponse:
     # DECLARATION -> EXTENSIONS
-    from extensions import _
-    _.__init__(request)
-    from extensions import database, post, random
-    
+    from website.extensions import Response; Response(request)
```

Ruff (0.11.9)

Multiple statements on one line (semicolon)

(E702)